src/client/components/Layout/Layout.js

```diff
@@ -53,3 +224,51 @@ Layout.propTypes = {
 };
 
 export default withRouter(Layout);
+
+/*
+<Sidebar.Pushable>
+                                            <Sidebar
+                                                as={Menu}
+                                                direction="top"
+                                                animation='overlay'
+                                                icon='labeled'
+                                                secondary
+                                                onHide={() => this.handleSectionSelector.bind(this, false )}
+                                                visible={sectionSelector}
+                                            >
+                                                <Segment>
+                                                    <Menu.Menu position='right'>
+                                                        <Menu.Item as='a'>
+                                                            <Icon name='home' />
+                                                            Home
+                                                        </Menu.Item>
+                                                        <Menu.Item as='a'>
+                                                            <Icon name='gamepad' />
+                                                            Games
+                                                        </Menu.Item>
+                                                        <Menu.Item as='a'>
+                                                            <Icon name='camera' />
+                                                            Channels
+                                                        </Menu.Item>
+                                                    </Menu.Menu>
+                                                </Segment>
+                                            </Sidebar>
+                                            <Sidebar.Pusher>
+                                                <Segment basic>
+
+                                                    {this.props.children}
+                                        
+                                                </Segment>
+                                            </Sidebar.Pusher>
+                                        </Sidebar.Pushable>
+*/
+
+
+/*
+                                                        
+                                                        {/* <Button.Group compact fluid basic> */
+                                                        /* <Button>Status</Button>
+                                                            <Button>Documentation</Button> */
+                                                            
+                                                        /* </Button.Group> */
+                                                        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/gh/ShipyardSuite/dashboard/pullRequest?prid=5673418)